server/models/articles.js

```diff
@@ -0,0 +1,50 @@
+/* eslint no-unused-vars:0 */
+
+
+module.exports = (sequelize, DataTypes) => {
+  const Articles = sequelize.define('Articles', {
+    id: {
```

Comment:
Thought we agreed to not use UUIDs? It should be consistent with what we have in the user model

Reply:
Actually, we agreed to use UUIDs but there was an oversight during the creation of User model.

Reply:
Feedback received